og/bump/action.yml

```diff
@@ -0,0 +1,33 @@
+name: bump
+description: |
+  Automatically bump the version file of the OG repo
+
+inputs:
+  base_branch:
+    description: The base branch of the PR.
+    required: true
+  pr_body:
+    description: OG formatted body of the PR description.
+    required: true
+  pr_title:
+    description: SC formatted title of the PR.
```

Comment:
I guess it's presently Shortcut... So we'd have to change this code of they started formatting things differently or we started using something else?

Reply:
This is repo is shared, and moving away from SC will invalidate this anyway, so I think it's okay to keep it here for clarity.